Sources/SwiftSyntaxBuilder/gyb_helpers/SyntaxBuildableWrappers.py

```diff
@@ -281,6 +281,15 @@ def element_in_collections(self):
         result.append(node.type())
     return result
 
+  def has_with_trailing_comma_trait(self):
+    """
+    Returns if this type has `WithTrailingComma` trait
+    """
+    for node in [SyntaxBuildableNode(node) for node in SYNTAX_NODES]:
+      if node.type() == self:
+        return node.node.traits and 'WithTrailingComma' in node.node.traits
+    return False
+
```

Comment:
Isn’t this equivalent to 

```suggestion
    return 'WithTrailingComma' in self.node.traits
```

Or am I missing something obvious?

Reply:
I don't think the type has info about the node
> AttributeError: 'SyntaxBuildableType' object has no attribute 'node'

Reply:
Oh, sorry, I thought this was defined on `SyntaxBuildableNode` not `SyntaxBuildableType`. In that case, it makes total sense 👍